it-tests/samples/sbt/sbt1.1/project/plugins.sbt

```diff
@@ -0,0 +1,6 @@
+
+////addSbtPlugin("io.shiftleft" % "sbt-dependency-graph" % "0.10.0-append-to-file3")
```

Comment:
What's the purpose of the already commented-out lines?

Reply:
It's a leftover, I didn't even intend to include this file